.github/workflows/rp2040_hal_examples.yml

```diff
@@ -5,14 +5,18 @@ env:
   TARGET: thumbv6m-none-eabi
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
         with:
           target: ${{ env.TARGET }}
       - name: Build
         run: cd ${PACKAGE} && cargo build
+      - name: Install picotool
+        run: sudo sh -c "apt-get install -y debian-archive-keyring && echo 'deb [signed-by=/usr/share/keyrings/debian-archive-keyring.gpg] http://deb.debian.org/debian testing main contrib non-free' >/etc/apt/sources.list.d/debian.list && apt update && apt-get install -y libxml2-utils picotool"
+      - name: Test picotool
+        run: picotool info ${PACKAGE}/target/${TARGET}/debug/binary_info_demo -t elf -a | tee /dev/stderr | grep -q "rp2040-hal Binary Info Example"
```

Comment:
What does tee'ing to /dev/stderr do here?

Reply:
`grep` consumes stdout and checks for some expected string - that's what makes the test fail or pass. But then, I also want to see the output of `picotool`, if only to understand the error cause if `grep` doesn't find the expected string. So `tee /dev/stderr` copies the output of `picotool` to stderr where it won't be affected by the following `grep`.